lib/netconf/local/genesis.json

```diff
@@ -194,7 +194,34 @@
           "current_epoch": "0",
           "current_epoch_start_time": "1970-01-01T00:00:00Z",
           "epoch_counting_started": false,
-          "current_epoch_start_height": "10"
+          "current_epoch_start_height": "1"
+        },
+        {
+          "identifier": "hour",
+          "start_time": "1970-01-01T00:00:00Z",
+          "duration": "1h",
+          "current_epoch": "0",
+          "current_epoch_start_time": "1970-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "1"
+        },
+        {
+          "identifier": "day",
+          "start_time": "1970-01-01T00:00:00Z",
+          "duration": "24h",
+          "current_epoch": "0",
+          "current_epoch_start_time": "1970-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "1"
+        },
+        {
+          "identifier": "week",
+          "start_time": "1970-01-01T00:00:00Z",
+          "duration": "168h",
+          "current_epoch": "0",
+          "current_epoch_start_time": "1970-01-01T00:00:00Z",
+          "epoch_counting_started": false,
+          "current_epoch_start_height": "1"
```

Comment:
Is current epoch-related information calculated based on the start time and duration of the epoch?

Reply:
Right. This is that [part](https://github.com/piplabs/story/blob/36a50a2ac21c044dc981c0ce5ef51176929d6975/client/x/epochs/keeper/abci.go#L31-L31)